src/components/MarsApp.css

```diff
@@ -3,9 +3,17 @@
 }
 
 .page-heading {
-  display: inline-block;
-  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
-  font-size: xx-large;
+  background: url(http://apod.nasa.gov/apod/image/0603/coma_misti.jpg) -80px -80px;
```

Comment:
Is this image licensed for reuse? If it isn't then we should find one that is and add it to our repo to avoid making calls to other websites 😁